about.html

```diff
@@ -10,54 +10,81 @@
   <link rel="stylesheet" href="css/index.css">
 
```

Comment:
-use the flexbox to its full potential (display / flex-direction / justify-content & align items) before going to padding / margin

- plan out your design ahead of time. this will help you understand where the parents and children should be and thus will help understand what flex properties to use

- stay away from using %'s when making vertical changes. % work well with horizontal spacing but can cause problems when appliying it vertically